### DISCUSSION TITLE
sh cross-env: not found

### DISCUSSION THREAD

**Discussion by jowin202** [danny-avila/LibreChat#4790] [Troubleshooting]
### What happened?

Hi,
I did the "usual installation" which I always do on test systems and got following error:
```
> LibreChat@v0.7.5. backend
> cross-env NODE_ENV=production node api/server/index.js

sh: cross-env: not found
```
(i typed the message from the screenshot in case of typos)

### Steps to Reproduce

`git clone https://github.com/danny-avila/LibreChat.git
`
prepare .env, docker-compose.override.yaml, librechat.yaml
in docker-compose.override file, I use the options for librechat config file and local build.

everything else is the same as in the repo
after docker-compose up, I got the cross-env: not found 

### Relevant log output

see error message above

### Screenshots

basically the given error message

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
try:

```bash
# stop running containers
docker compose down

# remove any outdated container images
docker rmi $(docker images | grep librechat | awk '{print $3}')

# get latest code changes
git pull

# pull latest iamges
docker compose pull

# try running
docker compose up
```

**Reply by jowin202** [reply]
I did, I created a new virtual machine for this. Ubuntu 24.04 + latest Docker Version

**Comment by jowin202** [comment]
Btw same result in Mac OS X
